BUILD.bazel

```diff
@@ -0,0 +1,7 @@
+exports_files(
```

Comment:
Let's keep consistent spacing here. How about a tab space of 2?

Reply:
👍 I've been using the bazel plugin for intellij which has enforced to me 4 spaces (and the configuration panel on the plugin is currently with a bug so I cannot change it 😄 )